content/css/concepts/typography/terms/font-face/font-face.md

```diff
@@ -25,10 +25,14 @@ The `@font-face` rule allows us to use custom fonts instead of just using "web-s
   src: url(' ') /* Link to font */;
 }
 ```
+There are different types of Font Formats that we use - WOFF,WOFF2(more compressed version of WOFF) EOT ,TTF and SVG.
 
-**Note:** The name of the file in which the font is stored, must end in a supported font format(e.g. **custom_font.woff2**, **custom_font.ttf**).
+**Note:** The name of the file in which the font is stored, must end in a supported font format(e.g. **custom_font.woff**,**custom_font.woff2**, **custom_font.ttf**,**custom_font.svg**,**custom_font.eot**).
+If we need our font to load faster then WOFF is the best(this format is supported in most of the browsers).
```

Comment:
I think font formats should be covered in greater detail in a different entry. Maybe in General File Formats (cc: @caupolicandiaz if you concur)

I'm thinking that it can go in a separate header ## Font Formats and be listed in table format. Then, we can link out to it here.